R/render.r

```diff
@@ -42,9 +42,12 @@ render_page <- function(pkg = ".", name, data, path = "", depth = NULL, quiet =
 
   data$footer <- pkgdown_footer(data, pkg)
 
+  # Dependencies for head
+  data$headdeps <- data_deps(pkg = pkg, depth = depth)
+
   # render template components
   pieces <- c(
-    "head", "navbar", "header", "content", "docsearch", "footer",
```

Comment:
header is actually empty

Reply:
but I now realize it might break current overriding templates? :thinking:

Reply:
Yeah, I think you'll need to leave as is.